src/Hl7.Fhir.Base/Rest/HttpClientRequester.cs

```diff
@@ -40,7 +41,7 @@ public HttpClientRequester(Uri baseUrl, FhirClientSettings settings, HttpClient
             _disposeHttpClient = false;
         }
 
-        public async Task<EntryResponse> ExecuteAsync(EntryRequest interaction)
+        public async Task<EntryResponse> ExecuteAsync(EntryRequest interaction, CancellationToken ct)
         {
             if (interaction == null) throw Error.ArgumentNull(nameof(interaction));
           
```

Comment:
Should we add in something here to not bother calling things if the cancellation token has already fired?
(for larger batches would then prevent going into the subsequent calls once the cancel is done)

Reply:
I have added token.ThrowIfCancellationRequested().